test/cpp/ext/csm/metadata_exchange_test.cc

```diff
@@ -328,6 +338,26 @@ TEST_P(MetadataExchangeTest, ClientDoesNotSendMetadata) {
             "unknown");
 }
 
+TEST_P(MetadataExchangeTest, VerifyCsmServiceLabels) {
+  Init(/*metric_names=*/{grpc::experimental::OpenTelemetryPluginBuilder::
+                             kClientAttemptDurationInstrumentName},
+       /*enable_client_side_injector=*/true,
+       /*add_service_labels_in_call=*/true);
```

Comment:
I'm not crazy about how the data we're injecting is hidden in the test framework, not something you can see just be reading the individual test.  Instead, how about we pass in a map of labels to inject, and then then the test can check that those values are present in the OT metrics data?

Reply:
Done.